target/hexagon/gen_helper_funcs.py

```diff
@@ -287,11 +287,14 @@ def main():
     hex_common.read_attribs_file(sys.argv[2])
     hex_common.read_overrides_file(sys.argv[3])
     hex_common.read_overrides_file(sys.argv[4])
+    if len(sys.argv) > 6:
```

Comment:
If this condition is false it means that `idef-parser` is not enabled, is that right?

Reply:
Yes exactly, it's not the most transparent way of doing things tho. If we pass 5 arguments it's disabled, if we pass 6 it's enabled. Any suggestion for improving this?

Reply:
Using `argparse`, but let's keep changes local for now. Maybe add a comment.

Reply:
Alright